addons/website/static/src/builder/plugins/customize_website_plugin.js

```diff
@@ -594,7 +609,10 @@ export class WebsiteConfigAction extends BuilderAction {
             }
             for (const item of action.selectableContext.items) {
                 for (const a of item.getActions()) {
-                    if (a.actionId === "websiteConfig") {
+                    if (
+                        a.actionId === "websiteConfig" ||
+                        a.actionId === "previewableWebsiteConfig"
+                    ) {
```

Comment:
Do you have a use case, where we want to use websiteConfig and previewableWebsiteConfig in the same groups ?